plugins/plugin-babel/test/plugin-babel.test.js

```diff
@@ -0,0 +1,117 @@
+const plugin = require('../plugin.js');
+const babel = require('@babel/core');
+const workerpool = require('workerpool');
+
+jest.mock('@babel/core');
+babel.transformFileAsync = jest.fn(() => Promise.resolve({code: 'code', map: 'map'}));
+
+/** jest mock above in worker will not work */
+jest.mock('workerpool');
```

Comment:
I've tried to move the the `jest.mock` calls to the top of the file and the tests passed. I'm a little confused about them being below the imports, too

```suggestion
jest.mock('@babel/core');
jest.mock('workerpool');

const plugin = require('../plugin.js');
const babel = require('@babel/core');
const workerpool = require('workerpool');
```

Otherwise this is good to go